packages/fp-behandling/src/fakta/components/person/panelBody/arbeidsforhold/PersonArbeidsforholdDetailForm.jsx

```diff
@@ -84,95 +94,98 @@ export const PersonArbeidsforholdDetailForm = ({
+                        value
+                        disabled={!aktivtArbeidsforholdTillatUtenIM}
+                      />
+                    </RadioGroupField>
+                  </Column>
+                </Row>
               )}
-            </RadioOption>
-            { skalKunneOverstyreArbeidsforholdTomDato(hasReceivedInntektsmelding, arbeidsforhold) && (
+            </BehandlingFormFieldCleaner>
+          </RadioOption>
+          { skalKunneOverstyreArbeidsforholdTomDato(hasReceivedInntektsmelding, arbeidsforhold) && (
             <RadioOption
               label={intl.formatMessage({ id: 'PersonArbeidsforholdDetailForm.ArbeidsforholdetErIkkeAktivt' })}
               value={false}
+              manualHideChildren
             >
-              <ArbeidsforholdAktivTomDatoPickerPanel
-                readOnly={readOnly}
-                fomDato={arbeidsforhold.fomDato}
-              />
+              <BehandlingFormFieldCleaner formName={PERSON_ARBEIDSFORHOLD_DETAIL_FORM} fieldNames={['overstyrtTom']}>
+                { skalBrukeUendretForhold === false && (
+                  <ArrowBox alignOffset={0}>
+                    <DatepickerField
+                      name="overstyrtTom"
+                      label={<FormattedMessage id="PersonArbeidsforholdDetailForm.ArbeidsforholdetAktivTomDato" />}
+                      validate={[required, hasValidDate, dateAfterOrEqual(arbeidsforhold.fomDato, arbeidsforholdTomDatoPickerErrorMsg)]}
+                      readOnly={readOnly}
+                    />
+                  </ArrowBox>
+                )}
+              </BehandlingFormFieldCleaner>
             </RadioOption>
-            )}
-            { !skalKunneOverstyreArbeidsforholdTomDato(hasReceivedInntektsmelding, arbeidsforhold) && (
+          )}
+          { !skalKunneOverstyreArbeidsforholdTomDato(hasReceivedInntektsmelding, arbeidsforhold) && (
             <RadioOption
               label={intl.formatMessage({ id: 'PersonArbeidsforholdDetailForm.ArbeidsforholdIkkeRelevant' })}
               value={false}
               disabled={isKildeAaRegisteret(arbeidsforhold)}
             />
-            )}
-          </RadioGroupField>
-        </BehandlingFormFieldCleaner>
+          )}
+        </RadioGroupField>
         <VerticalSpacer twentyPx />
         <BehandlingFormFieldCleaner formName={PERSON_ARBEIDSFORHOLD_DETAIL_FORM} fieldNames={['begrunnelse']}>
-          {showBegrunnelse
-          && (
-          <TextAreaField
-            name="begrunnelse"
-            label={intl.formatMessage({ id: 'PersonArbeidsforholdDetailForm.Begrunnelse' })}
-            validate={[required, maxLength400, hasValidText]}
-            maxLength={400}
-            readOnly={readOnly}
-          />
+          {showBegrunnelse && (
+            <TextAreaField
+              name="begrunnelse"
+              label={intl.formatMessage({ id: 'PersonArbeidsforholdDetailForm.Begrunnelse' })}
```

Comment:
Labels for TextAreaField treng heller ikkje intl:

```suggestion
          label={{ id: 'PersonArbeidsforholdDetailForm.Begrunnelse' }}
```

Då treng du ikkje ta inn intl i komponenten.